src/Altinn.App.Core/Models/Validation/InstantiationValidationResult.cs

```diff
@@ -12,6 +12,11 @@ public class InstantiationValidationResult
     /// </summary>
     public bool Valid { get; set; }
 
+    /// <summary>
+    /// The custom text key to use for the localized text.
+    /// </summary>
```

Comment:
Mention in summary that it is used for translation in backend if message is null, and in frontend anyway.